ddl/db_change_test.go

```diff
@@ -106,7 +111,12 @@ func (s *testStateChangeSuite) TestShowCreateTable(c *C) {
 			currTestCaseOffset++
 		}
 		if job.SchemaState != model.StatePublic {
-			result := tk.MustQuery("show create table t")
+			var result *testkit.Result
+			if job.Query[12:14] == "t2" {
```

Comment:
How about use `	t2 := testGetTableByName(c, tk.Se, "test", "t2")` then compare the `job.TableID == t2.Meta().ID`

Reply:
`[12:14]` is not extendable, and we shouldn't use magic number.

Reply:
Nice idea.